tests-legacy/Integration/Core/Localization/LocaleUsageTest.php

```diff
@@ -237,30 +238,31 @@ public function testItShouldFormatPrices($localeCode, $rawNumber, $currencyCode,
     protected static function installTestedLanguagePacks()
     {
         $countries = [
-            'us',
-            'jp',
-            'gb',
-            'de',
-            'fr',
-            'in',
-            'es',
-            'ca',
-            'cn',
-            'au',
-            'br',
-            'mx',
-            'ru',
-            'it',
-            'pl',
-            'bg',
-            'az',
+            'us' => 'en',
+            'jp' => 'ja',
+            'gb' => 'en',
+            'de' => 'de',
+            'fr' => 'fr',
+            'in' => 'hi',
+            'es' => 'es',
+            'ca' => 'ca',
+            'cn' => 'ug',
+            'au' => 'en',
+            'br' => 'pt',
+            'mx' => 'es',
+            'ru' => 'ru',
+            'it' => 'it',
+            'pl' => 'pl',
+            'bg' => 'bg',
+            'az' => 'az',
         ];
         $cacheDir = _PS_CACHE_DIR_ . 'sandbox' . DIRECTORY_SEPARATOR;
 
-        foreach ($countries as $country) {
+        foreach ($countries as $country => $iso) {
             $xmlContent = (new LocalizationWarmer(_PS_VERSION_, $country))
                 ->warmUp($cacheDir);
 
+            Language::checkAndAddLanguage($iso);
```

Comment:
I'm not so sure about this, shouldn't you use `$country` instead?

Reply:
Nope because of this: https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Language.php#L940

Reply:
If you want to get this: 
![image](https://user-images.githubusercontent.com/1462701/61518488-ef6f7f00-aa09-11e9-9026-cb7b21ae5feb.png)